core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -120,9 +120,12 @@ class PackagesResource(resolver: ExternalResolverClient, db : Database)
 
   val route = pathPrefix("packages") {
     get {
-      complete {
-        NoContent
-        //Packages.list
+      parameters('regex.?) { (regex) =>
```

Comment:
The Refined library that we are using for validation of input can validate regexes, by the way. See: https://github.com/fthomas/refined#more-examples , hooking our validation machinery up to parameters rather than put/post/get might require a little bit of work though.